data/templates/dnsmasq/plain/resolv.dnsmasq.conf

```diff
@@ -0,0 +1,26 @@
+# According to dnsmasq's doc, dnsmasq will automatically 
+# choose the fastest-to-answer resolver as primary 
+# (if strict-order is disabled)
+
+# List taken from
+# http://diyisp.org/dokuwiki/doku.php?id=technical:dnsresolver
+
+# (FR) FDN
+80.67.169.12
```

Comment:
Is it the proper format ? I thought it must be like `nameserver 11.22.33.44` according to dnsmasq documentation: "For the format of this file see resolv.conf(5)."